app/components/visualizer/main-viz/templates/Chart.jsx

```diff
@@ -0,0 +1,42 @@
+import DynamicPieChart from "./DynamicPieChart.js";
+
+class Chart extends React.Component {
+  componentDidMount() {
+    let el = React.findDOMNode(this);
+
+    $.getJSON("blobs/deptGrades.json", (data) => {
+      this.chart = new DynamicPieChart({
+        id: "_id",
+        data: data,
+        element: "#main-viz",
+        height: 512,
+        width: 1024,
```

Comment:
yeah i agree it is bad. we actually need to dynamically size the svg, which as of now is an unsolved issue. this will invariably need to be refactored, prob outside the scope of this (already large) PR